### PR TITLE
Fix #3737 - Popup search for documents in mail compose not working on 7.8.x LTS

### LINKED ISSUE #3737: popup search for documents in mail compose not working
#### Issue
The popup search filter for documents in suitecrm when you compose a mail is not working an always list all the documents in suitecrm.

#### Expected Behavior
The listed documents should match with the filters applied by the user.

#### Actual Behavior
All documents in suitecrm are listed.

#### Possible Fix
???? not a programer

#### Steps to Reproduce
1. Be sure at least you have one document created
2. Compose an email
3. Click on the right bar for attachments an templates
4. ADD DOCUMENT (From SuiteCRM)
5. Click in arrow
6. Filter with a name document you haven't ('kkkkkkkkkkk')
7. SEARCH

areadetrabajo1626

#### Context
Actually we have thousand of documents and user can't find the documents they want to attach. They must go to the Documents module, search the document, download lo local and attach it in the mail using the ADD FILE (From local System). No way to attach files when they are on mobile.

#### Your Environment
* SuiteCRM Version used: 7.8.5
* Browser name and version: Versión 58.0.3029.110 (64-bit) (e.g. Chrome Version 51.0.2704.63 (64-bit))
* Environment name and version:  5.6.33
* Operating System and version: 16.04

### PR DESCRIPTION
**Pull request by sanchezfauste**
The popup search filter for documents in suitecrm when you compose a mail is not working an always list all the documents in suitecrm.

## Description

See #3737 and #3795

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [x] My code follows the code style of this project found [here](https://docs.suitecrm.com/community/contributing-code/coding-standards/).
- [ ] My change requires a change to the documentation.
- [x] I have read the [**How to Contribute**](https://docs.suitecrm.com/community/contributing-code/) guidelines.

### CODE REVIEW

File: include/Popups/DocumentPopupPicker.php [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+if (!defined('sugarEntry') || !sugarEntry) {
+    die('Not A Valid Entry Point');
+}
+
+require_once('include/Popups/Popup_picker.php');
+
+class DocumentPopupPicker extends Popup_Picker
+{
+    public function _get_where_clause()
```
Comment (PedroErnst) [inline include/Popups/DocumentPopupPicker.php]: Please include PHP Docs for the method and the class

**Comment by sanchezfauste** [comment]
Hi @PedroErnst, I have updated the code applying your requested changes. Is PHP Docs correct?

**Comment by PedroErnst** [comment]
Looks fantastic!

**Comment by Dillon-Brown** [comment]
Assessed

**Pull request merged**